### ISSUE TITLE
Typo in electronicFormatIBAN function name

### ISSUE THREAD

**Issue by Dam-Buty** [Simplify/ibantools#4]
### Expected behavior

```
const ibantools = require('ibantools');
ibantools.electronicFormatIBAN('DE59760200700019367282');
```

### Actual behavior

`ibantools.electronicFormatIBAN is not a function`

### Steps to reproduce the behavior

The issue comes from a slight typo in the module's source, where `electronicFormatIBAN ` is spelt `electonicFormatIBAN ` (with an `r` missing in `electronic`)

**Comment by Dam-Buty** [comment]
Also, there is an error in the code examples of the documentation of `friendlyFormatIBAN`

![image](https://user-images.githubusercontent.com/1461760/31389920-c306967c-add2-11e7-9cce-d5d5c6b4eb03.png)

**Comment by Simplify** [comment]
@Dam-Buty Thanks for reporting this :)

Problems are fixed now and version 1.5.0 is released. Documentation is updated too on gh-pages branch.
